src/Configuration/Config.UserSecrets/test/Microsoft.Extensions.Configuration.UserSecrets.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.0;net472</TargetFrameworks>
```

Comment:
just curious why user secrets gets a net472?

Reply:
I added this to the tests so that it will exercise the netstandard2.0 binaries. I see this pattern elsewhere in our tests in this repo.

Reply:
FYI @davidfowl \ @rynowak. Since we're cross-compiling these binaries, they're being tested on net4x to get coverage for the .NETStandard codepath. 

@JunTaoLuo does this need to do anything special to not run on xplat?

Reply:
No, we don't need any additional logic for that. It's already handled for us, which is why most of the tests in this repo targets both TFMs.